### PR TITLE
Replacement of @extends

### LINKED ISSUE #781: Remove all use of `@extends` from Vocabulary
## Description

Extends, particularly of bulma helper classes, can cause issues in scss projects using vocabulary. To build properly, these projects will need to import all bulma helpers themselves, which is concerning because it requires them to manage bulma on their own, which could result in them using a different version than vocabulary.

Here are all uses of `@extends` in Vocabulary: https://github.com/creativecommons/vocabulary/search?q=%22%40extend%22&type=code

They should be replaced like this:

```diff
- @extends .padding-top-normal
+ padding-top: $space-normal
```

and so on.

### PR DESCRIPTION
**Pull request by neeraj-2**
## Fixes

Fixes #781 by @zackkrida 

## Description

This PR replaces the use of  `@extends ` in .scss files of **Vocabulary Package**

## Checklist
 
- [X] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [X] My pull request targets the *default* branch of the repository (`main` or `master`).
- [X] My commit messages follow [best practices][best_practices].
- [X] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [X] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: packages/vocabulary/src/styles/button.scss [hunk 1]
```diff
@@ -187,8 +187,14 @@ $button-text-hover-color: $color-tomato;
   }
 
   &.is-text {
+
```
Comment (nimishbongale) [inline packages/vocabulary/src/styles/button.scss]: Maybe this extra line space can be removed?
```suggestion
```

File: packages/vocabulary/src/styles/button.scss [hunk 2]
```diff
@@ -187,8 +187,14 @@ $button-text-hover-color: $color-tomato;
...
+    padding: none;
+    height: auto !important;
+    line-height: normal;
+
     font-family: $family-source-sans-pro;
-    @extend .is-paddingless;
+   
+
```
Comment (nimishbongale) [inline packages/vocabulary/src/styles/button.scss]: Here as well, to match the spacing in the rest of the sections.
```suggestion
```

File: packages/vocabulary/src/styles/card.scss [hunk 3]
```diff
@@ -289,37 +297,37 @@ $card-header-shadow: none;
...
+                        padding-left: $space-small;
+                        padding-top: $space-smaller;
+                        color: $color-forest-green;
                     }
                 }
                 .content {
-                    @extend .margin-top-bigger;
-                    @extend .is-paddingless;
+                    margin-top: $space-bigger;
+                    padding: none;
```
Comment (obulat) [inline packages/vocabulary/src/styles/card.scss]: Padding should be set to 0.

**Review by obulat — APPROVED** [review]
Looks good, great job, @neeraj-2 !
Only one change requested inline (padding 0, not none).

File: packages/vocabulary/src/styles/card.scss (hunk 3 above)
Comment (neeraj-2) [reply packages/vocabulary/src/styles/card.scss]: thanks, @obulat for reviewing it, ok I will make the required changes.

File: packages/vocabulary/src/styles/notification.scss [hunk 4]
```diff
@@ -87,7 +87,7 @@
   }
   &.is-compact {
     .notification-container {
-      @extend .padding-vertical-normal;
+      padding-top: $space-normal;
```
Comment (zackkrida) [inline packages/vocabulary/src/styles/notification.scss:90]: Wouldn't `.padding-vertical-normal` set the top _and bottom_ padding?

File: packages/vocabulary/src/styles/image.scss [hunk 5]
```diff
@@ -71,8 +71,8 @@
 
   &.is-compact {
     .image-icons {
-      @extend .is-paddingless;
-      @extend .padding-horizontal-small;
+      padding: 0;
+      padding-left: $space-small;
```
Comment (zackkrida) [inline packages/vocabulary/src/styles/image.scss:79]: Wouldn't `.padding-horizontal-small` set the left _and right_ padding?

**Review by zackkrida — CHANGES_REQUESTED** [review]
The one thing I want to confirm: There's several places where `.padding-horizontal-*` was replaced with `padding-left: *`. Wouldn't that need to set `padding-right: *` too? Same for `padding-horizontal-*` with top and bottom. 

I might be mistaken/missing something.

File: packages/vocabulary/src/styles/image.scss (hunk 5 above)
Comment (obulat) [reply packages/vocabulary/src/styles/image.scss:79]: Definitely! I missed that.

Comment (neeraj-2) [reply packages/vocabulary/src/styles/image.scss:79]: I will update

**Review by zackkrida — APPROVED** [review]
Great!

**Comment by neeraj-2** [comment]
> Great!

Thanks, @zackkrida, and @obulat 😊!

**Pull request merged**